src/coreclr/vm/ceeload.h

```diff
@@ -615,6 +611,7 @@ class Module : public ModuleBase
 
 private:
     PTR_CUTF8               m_pSimpleName; // Cached simple name for better performance and easier diagnostics
+    const WCHAR*            m_path;        // Cached path for easier diagnostics
```

Comment:
Are we also trying to avoid exposing SString for the cdac? I would otherwise expect the runtime to use SString more pervasively rather than raw char pointers.

Reply:
We need to be aware of lifetime. In general, an `SString` holds has its own memory, and a WCHAR* assumes something else is keeping it alive. It looks like this change attempts to avoid allocating a new copy of the string data. In general, I'm a big fan of avoiding `SString` where possible, as it has all sorts of weird costs, and if you pass it around, you could accidentally end up modifying it or something. `SString` generally works better as a stack type. However, it does make looking at the string from across the DAC a bit easier since SString keeps counts and such around. Looking at @elinor-fung 's commit, I'm not convinced that it would successfully actually read a string, you might need to do extra shenanigans to make that work across the DAC boundary.

Reply:
The `SString` data type should not be exposed in the cDAC contract.